operations/system_layout_service/Load_SLS_Database_with_Dump_File.md

```diff
@@ -21,17 +21,13 @@ The System Layout Service \(SLS\) database has been dumped. See [Dump SLS Inform
     }
     ```
 
-2.  Load the dump file into SLS.
+1. (`ncn-mw#`) Load the dump file into SLS.
 
-    This will upload and overwrite the current SLS database with the contents of the posted file, as well as update the Vault with the encrypted credentials. The private key that was used to generate the SLS dump file is required.
+    This will upload and overwrite the current SLS database with the contents of the posted file.
 
     ```bash
     curl -X POST \
```

Comment:
Can't this `curl` command be replaced with the craycli now? That now that https://github.com/Cray-HPE/craycli/pull/58 has merged?

Reply:
If going that route, be sure to add a prerequisite about the Cray CLI being configured. Something like:

```markdown
- The Cray Command Line Interface is configured. See [Configure the Cray CLI](../configure_cray_cli.md).
```

Reply:
These changes have been made